src/tests/src/libs/antares/study/test_study.cpp

```diff
@@ -291,4 +291,20 @@ BOOST_AUTO_TEST_CASE(version_parsing)
     BOOST_CHECK(!v.fromString("4.5"));
     BOOST_CHECK(v == StudyVersion::unknown());
 }
+
+BOOST_FIXTURE_TEST_CASE(check_filename_limit, OneAreaStudy)
```

Comment:
Tests are normally more atomic.
Here, several things are tested in one test.
The results is that this new test has a more complex setting than it should have : it uses a fixture (which creates a **study**) and it creates another empty **study**.

This test checks 4 things.
We should create 4 smaller tests instead, one for each case, use the fixture only if a test requires it, and name these smaller tests more specifically, for clarity.